src/ContentHubClient.php

```diff
@@ -1056,7 +1056,7 @@ public function __call($method, $args)
     {
         try {
             if (strpos($args[0], '?')) {
-                [$uri, $query] = explode('?', $args[0]);
```

Comment:
Do we have PHP version issue with this?

Reply:
Yes, minimal PHP version is 5.6
https://github.com/acquia/content-hub-php/blob/2.x/composer.json#L17